client/components/SingleProduct.js

```diff
@@ -46,12 +47,12 @@ class SingleProduct extends Component {
   componentDidMount() {
     const productId = this.props.match.params.productId
     this.props.fetchInitialProduct(productId)
+    this.props.fetchReviews(productId)
   }
 
   render() {
-    const {product, classes} = this.props
-    console.log(product)
-    while (!product.id) {
+    const {product, reviews, classes} = this.props
```

Comment:
we need to consider what happens if product gets deleted...don't want the loading page to load forever